pages/common/rm.md

```diff
@@ -10,6 +10,10 @@
 
 `rm -r {{/path/to/folder}}`
 
+- Remove file/directory without prompt:
```

Comment:
Um... `-r` can use with file, but doesn't make sense when only with file, but I always use `-rf` for file or directory, because I only need to remember 1 command, or when I use `*`.